sdks/java/core/build.gradle

```diff
@@ -117,6 +132,110 @@ task packageTests(type: Jar) {
   classifier = "tests"
 }
 
+class ValidatesRunnerConfig {
+  // Runner name prefix
+  String runner
+  // List of test categories to exclude from this task. Optional.
+  List<String> excludes
+  // Pipeline options command line arguments.
+  Map<String, String> pipelineOptions
+  // Configuration to use for test runtime classpath.
+  FileCollection configuration
+  // Additional system properties to be set for tests. Optional.
+  Map<String, String> systemProperties
+}
+
+def createValidatesRunner(Map m) {
```

Comment:
`createValidatesRunner` -> `createValidatesRunnerTask`

Also add a comment as to what the method does, what is the expected input parameter, ...